requirements/base.in

```diff
@@ -17,3 +17,8 @@
 # under the License.
 #
 -e file:.
+
+# pinning these two dependencies that pip-compile-multi complained about
```

Comment:
@mistercrunch this feels somewhat icky and will likely corner us in the future. Have you tried simply running `pip-compile-multi --upgrade` to give it a fighting chance of finding an eligible set of frozen dependencies.

Reply:
oh didn't know that was an option, let me re-try

Reply:
form `pip-compile-multi --help`
```
  --upgrade / --no-upgrade        Upgrade package version (default true)
```
Trying to run it I get the same error as above